Description: Fix type inconsitencies.
 mapfile.c line 65: error: conflicting types for variable `msyystring_icase'
 old definition in module `maplexer' file maplexer.l line 51
 signed int
 new definition in module `mapfile' file mapfile.c line 65
 char
Author: Michael Tautschnig <mt@debian.org>
Bug-Debian: https://bugs.debian.org/812685
Forwarded: https://github.com/mapserver/mapserver/pull/5227
Applied-Upstream: https://github.com/mapserver/mapserver/commit/278a280411130f2a0521afb222d7780d2581c816

--- a/mapfile.c
+++ b/mapfile.c
@@ -62,7 +62,7 @@ extern char *msyystring;
 extern char *msyybasepath;
 extern int msyyreturncomments;
 extern char *msyystring_buffer;
-extern char msyystring_icase;
+extern int msyystring_icase;
 
 extern int loadSymbol(symbolObj *s, char *symbolpath); /* in mapsymbol.c */
 extern void writeSymbol(symbolObj *s, FILE *stream); /* in mapsymbol.c */
